<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>900</width>
       <height>650</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(70, 70, 70);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QTableWidget" name="tableWidget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(128, 128, 128);</string>
        </property>
        <column>
         <property name="text">
          <string>orders</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>cost</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="delete_button">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:16px;
border-color: rgb(199, 98, 50);
background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Sil</string>
          </property>
          <property name="icon">
           <iconset theme="backwardbend.png">
            <normaloff>../side_projects/StockManager</normaloff>../side_projects/StockManager</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_button">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:16px;
border-color: rgb(199, 98, 50);
background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Siparis Iptal</string>
          </property>
          <property name="icon">
           <iconset theme="backwardbend.png">
            <normaloff>../side_projects/StockManager</normaloff>../side_projects/StockManager</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ok_button">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:16px;
border-color: rgb(199, 98, 50);
background-color: rgb(21, 115, 215);
color:rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>Siparisi Onayla</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(190,134,80)</string>
          </property>
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="check">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(190,134,80)</string>
          </property>
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="order_id_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>ORDER</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="order_id">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>0001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" rowspan="2">
       <layout class="QGridLayout" name="urunbutonlari_layout">
        <item row="0" column="0">
         <widget class="QPushButton" name="b_filtercoffee">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>61</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:16px;
border-color: rgb(199, 98, 50);
background-color:rgb(160, 140, 110);
color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Filter Coffee</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="b_espresso">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>61</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:16px;
border-color: rgb(199, 98, 50);
background-color:rgb(160, 140, 110);
color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Espresso</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="b_americano">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>61</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:16px;
border-color: rgb(199, 98, 50);
background-color:rgb(160, 140, 110);
color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Americano</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="b_latte">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>61</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:16px;
border-color: rgb(199, 98, 50);
background-color:rgb(160, 140, 110);
color:rgb(0, 0, 0)
</string>
          </property>
          <property name="text">
           <string>Latte</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
